nautobot/ipam/utils/__init__.py

```diff
@@ -271,5 +273,5 @@ def retrieve_interface_or_vminterface_from_request(request):
     try:
         obj = interface_model.objects.restrict(request.user, "change").get(id=interface_id)
         return obj, None
-    except interface_model.DoesNotExist:
+    except (interface_model.DoesNotExist, ValidationError):
```

Comment:
How do we trigger a ValidationError here and do we need a test case for it?

Reply:
I did it manually by passing in an invalid (non-UUID) `interface` or `vminterface` query parameter (`?interface=123foobar`). I don't think it's important enough to need a test case for, personally.